lib/server/hqqresolver.py

```diff
@@ -8,18 +8,45 @@
 # *  http://www.gnu.org/copyleft/gpl.html
 # *
 # *
-# *  based on https://gitorious.org/iptv-pl-dla-openpli/ urlresolver
+# *  original based on https://gitorious.org/iptv-pl-dla-openpli/ urlresolver
+# *  update based on https://github.com/LordVenom/
 # */
 from StringIO import StringIO
 import json
 import re
-import util
 import base64
 import urllib
+import urllib2
 
 __name__ = 'hqq'
+UA = 'Mozilla/6.0 (Windows; U; Windows NT 5.1; en-GB; rv:1.9.0.5) Gecko/2008092417 Firefox/3.0.3'
 
+def request(url, headers={}):
+    print('request: %s' % url)
+    req = urllib2.Request(url, headers=headers)
+    req.add_header('User-Agent', UA)
+    try:
+        response = urllib2.urlopen(req)
+        data = response.read()
+        response.close()
+    except urllib2.HTTPError, error:
+        data=error.read()
+    print('len(data) %s' % len(data))
+    return data
 
+def post(url, data, headers={}):
+    postdata = urllib.urlencode(data)
+    req = urllib2.Request(url, postdata, headers)
+    req.add_header('User-Agent', UA)
+    try:
+        response = urllib2.urlopen(req)
+        data = response.read()
+        response.close()
+    except urllib2.HTTPError, error:
+        data=error.read()
+    print('len(data) %s' % len(data))
+    return data
+    
```

Comment:
Why have you copied the code from `util` ? I'm not OK with reimplementing these functions.